src/CommentSold/Api/Clients/Rest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CommentSold\Api\Clients;
+
+use CommentSold\Api\Exception\CommentSoldException;
+use CommentSold\Api\Exception\InvalidResponseException;
+use CommentSold\Api\Response;
+use GuzzleHttp\Client;
+
+class Rest
+{
+    private const TOKEN_SERVICE_URL = 'https://tokens.cs-api.com';
+    private const API_URL = 'https://openapi.commentsold.com/v1';
```

Comment:
Once we have a sandbox setup I plan on having a `->useSandbox()` (or something like that) on the client so you can specify using the sandbox or production. That way there is no real change to client code and I can update the internals to use a getter instead of the constants directly.